src/components/Disputes/hooks/useJurorDraftQuery.js

```diff
@@ -0,0 +1,18 @@
+import { useQuery } from 'urql'
+import { JurorDrafts } from '../../../queries/disputes'
+import { reduceDispute } from '../reducer'
+
+export default function useJurorDraftQuery(juror) {
+  const [result] = useQuery({
+    query: JurorDrafts,
+    variables: { id: juror },
+  })
+
+  return !result.fetching && !result.error
+    ? result.data.juror
+      ? result.data.juror.drafts.map(draft =>
+          reduceDispute(draft.round.dispute)
+        )
+      : []
+    : []
```

Comment:
Just a minor style thing again, but what would you think about returning early to avoid nested ternary operators?

```jsx
if (result.fetching || result.error) {
  return []
}

return result.data.juror
  ? result.data.juror.drafts.map(draft =>
      reduceDispute(draft.round.dispute)
    )
  : []

Reply:
Agree, and in the next PR i moved some folders outside the Dispute component Folder like the hooks folder that is in the wrong place and the types